api/Infrastructure/Mappings/CaseMapping.cs

```diff
@@ -27,7 +27,8 @@ public void Register(TypeAdapterConfig config)
             .Map(dest => dest.Reason, src => src.NextApprReason)
             .Map(dest => dest.PartId, src => src.ProfPartId)
             .Map(dest => dest.DueDate, src => src.NextApprDt)
-            .Map(dest => dest.AgeInDays, src => src.CaseAgeDays);
+            .Map(dest => dest.AgeInDays, src => src.CaseAgeDays)
+            .Map(dest => dest.PhysicalFileId, src => src.JustinNo ?? src.PhysicalFileId);
```

Comment:
Just to confirm that this either JustinNo is populated or PhysicalFileId is populated? otherwise perhaps we could just have independent fields for both.